packages/expo/src/Updates/Updates.ts

```diff
@@ -1,9 +1,43 @@
 import { UnavailabilityError } from '@unimodules/core';
+import Constants from 'expo-constants';
 import { EventEmitter, EventSubscription } from 'fbemitter';
 import DeviceEventEmitter from 'react-native/Libraries/EventEmitter/RCTDeviceEventEmitter';
 
 import ExponentUpdates from './ExponentUpdates';
 
+type Manifest = typeof Constants.manifest;
+
+type UpdateCheckResult =
+  | {
+      isAvailable: false;
+      manifest: undefined;
+    }
+  | {
+      isAvailable: true;
+      manifest: Manifest;
+    };
+
+type UpdateFetchResult =
```

Comment:
sorry, this should be

```ts
type UpdateFetchResult = {
  isNew: boolean;
  manifest: Manifest;
}
```

according to the docs: https://docs.expo.io/versions/latest/sdk/updates/#updatesfetchupdateasyncparams